### DISCUSSION TITLE
No tests found, maybe i made a mistake with configuration?

### DISCUSSION THREAD

**Discussion by l3Black** [rcasia/neotest-java#137] [Q&A]
Hello!
I'm using AstroNvim with java community pack (which includes preconfigured nvim-jdtls and java-test), and try to configure neotest, but always I see the "No tests found" notification.
my neotest config
![image](https://github.com/user-attachments/assets/59f6b990-4d26-4628-bfbb-2ee6fae59a10)
my nvim-jdtls config (only java runtimes, other options by default from java community pack)
![image](https://github.com/user-attachments/assets/176a2af6-5a5c-41d6-8b7e-8cc7f55ab08f)
my project is just spring initializer example
![image](https://github.com/user-attachments/assets/29527246-c793-4fdc-8885-04c835659292)
If I run :lua require('jdtls').test_nearest_method() it works fine
![image](https://github.com/user-attachments/assets/2f417c2f-2ac6-4a32-a3fd-18c0f30392be)
my tests summary 
![image](https://github.com/user-attachments/assets/f0ed56b8-0f36-4664-93ee-5c5f43dc20b2)
Where's I wrong with neotest configuration?

**Comment by rcasia** [comment]
Have you tried to run `:TSInstall java`? Test detection is done with Treesitter, so the problem should be related with that.

**Reply by l3Black** [reply]
Yes, I have tried 
![image](https://github.com/user-attachments/assets/d4a7d0ea-d8d1-4408-ac32-a01b371a8747)

**Comment by Qualus** [comment]
Hello! Just stumbled on this one and I resolved a similar issue.

First, increase the log level to debug, you might have an error and that message might be misleading.

I did and it turned out the issue was I was using neovim 0.95 and you need 10 to use neotest-java
